backend/bittan/bittan/services/swish/__init__.py

```diff
@@ -0,0 +1,163 @@
+from django.utils.log import logging
```

Comment:
use default logging module for continuity

Reply:
9e7e1d5cbd652c4674ea7b42ecd94daa4d9ed38d